src/css/profile/mobile/themes/light.variables.less

```diff
@@ -16,6 +16,9 @@
     @overlay-opacity: 65%;
     @popup-background-color: @background-area-color;
     @popup-text-color: #505050;
+    @popup-2level-description: #8f8f8f;
+    @popup-listview-1line: #252525;
+    @popup-listview-2line: #909090;
```

Comment:
the same: @text-secondary-color

Reply:
I know what the same @text-secondary-color.
However, the color is differently displayed in the specification document according to the lignt and dark theme. I think it is good to put it intentionally.